lib/ui/pages/account_dashboard.dart

```diff
@@ -0,0 +1,351 @@
+                    ),
+                    trailing: TitleText(
+                      text: t.payeeName,
+                      fontSize: 14,
+                    ),
+                  ),
+                ),
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(
+                    20.0,
+                    0,
+                    20,
+                    0,
+                  ),
+                  child: ListTile(
+                    contentPadding: const EdgeInsets.symmetric(),
+                    title: const TitleText(
+                      text: 'Date',
+                      fontSize: 14,
+                    ),
+                    // subtitle: Text(selectedAccount.accountNumber),
+                    trailing: TitleText(
+                      text: t.date,
+                      fontSize: 14,
+                    ),
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ],
+      ),
+    ));
+  }
+
+  Widget _transaction(Transaction t) {
+    final IconData icon = t.status == Status.SUCCESSFUL
+        ? Icons.check_circle_outline
+        : (t.status == Status.PENDING
+            ? Icons.hourglass_empty
+            : Icons.error_outline);
+    final Color textColor = t.status == Status.SUCCESSFUL
+        ? Colors.green
+        : (t.status == Status.PENDING ? LightColor.yellow2 : Colors.red);
+
+    return ListTile(
+      onTap: () {
+        _showTransactionDetailBottomSheet(t);
+      },
+      leading: Container(
+        height: 50,
+        width: 50,
+        decoration: BoxDecoration(
+          color: textColor,
+          borderRadius: const BorderRadius.all(
+            Radius.circular(
+              10,
+            ),
+          ),
+        ),
+        child: Icon(icon, color: Colors.white),
+      ),
+      contentPadding: const EdgeInsets.symmetric(),
+      title: TitleText(
+        text: t.payeeName,
+        fontSize: 14,
+      ),
+      subtitle: Text(
+        t.date,
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(body: SafeArea(
+      child: SingleChildScrollView(
+        child: Container(
+          padding: const EdgeInsets.symmetric(
+            horizontal: 20,
+          ),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              const SizedBox(
+                height: 35,
+              ),
+              _appBar(),
+              const SizedBox(
+                height: 50,
+              ),
+              const TitleText(
+                text: 'Operations',
+              ),
+              const SizedBox(
+                height: 10,
+              ),
+              _operationsWidget(),
+              const SizedBox(
+                height: 40,
+              ),
+              const TitleText(
+                text: 'Transactions',
```

Comment:
```suggestion
                text: 'Transactions'
```

Same here, I think this is simple enough to not use trailing commas

Reply:
applies to the other parts of the code before this